testing/sdk_metrics.py

```diff
@@ -136,7 +163,19 @@ def get_metrics(package_name, service_name, pod_name, task_name):
         ),
         retry=False,
     )
-    app_json = json.loads(app_response.text)
+    app_response.raise_for_status()
+    app_json = app_response.json()
+
+    if "dimensions" not in app_json:
+        log.error("Expected key '%s' not found in app metrics: %s", "dimensions", app_json)
+        raise Exception("Expected key 'dimensions' not found in app metrics: {}")
+
+    if "task_name" not in app_json["dimensions"]:
+        log.error(
+            "Expected key '%s' not found in app metrics: %s", "dimensions.task_name", app_json
+        )
+        raise Exception("Expected key 'dimensions' not found in app metrics: {}")
```

Comment:
Change error messages. Expected key `task_name` ..... instead of `dimensions`.

Reply:
3fcf4de